srsue/hdr/stack/rrc_nr/rrc_nr_config.h

```diff
@@ -40,6 +40,7 @@ struct rrc_nr_args_t {
   srsran_subcarrier_spacing_t ssb_scs;
   std::string                 log_level;
   uint32_t                    log_hex_limit;
+  int                         sdu_fuzzed_bits;
```

Comment:
Avoid using generic (old C-style) types. I'd use a `uin32_t` instead, since the size of an `int` is only vaguely specified in the C/C++ standards.

Reply:
I've changed the value to uint32_t for both LTE and NR. Thanks for the great suggestion.